config/default.yml

```diff
@@ -442,7 +442,7 @@ RSpec/ExpectOutput:
 
 RSpec/FilePath:
   Description: Checks that spec file paths are consistent and well-formed.
-  Enabled: true
+  Enabled: false
```

Comment:
When using `inherit_gem` I get the following error:

    Error: The `RSpec/FilePath` cop has been split into `RSpec/SpecFilePathFormat` and `RSpec/SpecFilePathSuffix`.
    (obsolete configuration found in /Users/bquorning/.gem/ruby/2.7.8/gems/rubocop-rspec-2.24.0/config/default.yml, please update it)

```yaml
require:
  - rubocop-rspec

inherit_gem:
  rubocop-rspec: config/default.yml
```

Reply:
@bquorning Shouldn’t you also inherit the obsoletion config if you inherit the default config? I’m looking at this doc https://docs.rubocop.org/rubocop/configuration.html#inheriting-configuration-from-a-dependency-gem and it states that it is possible to inherit from several config files:
‘’’
inherit_gem:
  my-shared-gem:
    - default.yml
    - strict.yml
‘’’
Or am I missing something? What is your use cas?

Reply:
Please disregard my comment above, it seems that the default config already loaded the obsoletion config.
Why would you inherit from the default config? Isn’t it the same as ‘require: rubocop-rspec’?

Reply:
This is a project where I use StandardRb’s configuration of RuboCop, along with other plugins such as rubocop-rspec. [Their documentation](https://github.com/standardrb/standard#running-standards-rules-via-rubocop) gives an example of how to do this setup:

```
require:
  - standard
  - standard-custom
  - standard-performance
  - rubocop-performance

inherit_gem:
  standard: config/base.yml
  standard-custom: config/base.yml
  standard-performance: config/base.yml
```

I guess I got carried away when adding lines to `inherit_gem`. When I remove `rubocop-rspec` from that section, it works perfectly 👍🏼